pkg/resources/external_table_grant.go

```diff
@@ -216,3 +215,56 @@ func DeleteExternalTableGrant(d *schema.ResourceData, meta interface{}) error {
 	}
 	return deleteGenericGrant(d, meta, builder)
 }
+
+// UpdateExternalTableGrant implements schema.UpdateFunc
+func UpdateExternalTableGrant(d *schema.ResourceData, meta interface{}) error {
+	// for now the only thing we can update are roles or shares
+	// if nothing changed, nothing to update and we're done
+	if !d.HasChanges("roles", "shares") {
```

Comment:
since everything else is marked as forcenew, this code will actually never run. but at least it makes it clear what future work needs to be done

Reply:
@sfc-gh-swinkler The ForceNew should be removed that's why the Update was implemented